src/components/BlockList/BlockList.tsx

```diff
@@ -60,29 +60,41 @@ function BlockList({ blocks }: any) {
             }
 
             // Calculate difference of block timestamp from that of parent.
-            const timeDifferenceFromParent = (index === sortedBlocks.length-1) ? 0 : hexToNumber(b.timestamp) - hexToNumber(sortedBlocks[index+1].timestamp);
+            const timeDifferenceFromParent = (index === sortedBlocks.length - 1) ? 0 : hexToNumber(b.timestamp) - hexToNumber(sortedBlocks[index + 1].timestamp);
 
             return (
               <TableRow key={b.number} style={authorHashStyle}>
                 <TableCell style={rightPaddingFix}>
-                <Typography>
-                <Link
+                  <Typography>
+                    <Link
+                      component={({ className, children }: { children: any, className: string }) => (
+                        <RouterLink className={className} to={`/address/${b.miner}`} >
+                          {children}
+                        </RouterLink>
+                      )}>
+                      {authorHashShort}
+                    </Link>
+                    &nbsp;<sup>{hexToString(b.extraData).substring(0, 20)}</sup>
```

Comment:
Eventually we might want to refactor this address/+extradata format out to a configurable/standard, eg. substring length, abbreviations, sanitization, ...

Reply:
agreed. lets make a ticket for that

Reply:
https://github.com/etclabscore/expedition/issues/234